src/WrappedMToken.sol

```diff
@@ -545,9 +545,10 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
      * @dev   Wraps `amount` M from `account_` into wM for `recipient`.
      * @param account_   The account from which M is deposited.
      * @param recipient_ The account receiving the minted wM.
-     * @param amount_    The amount of M deposited and wM minted.
+     * @param amount_    The amount of M deposited.
+     * @param wrapped_   The amount of wM minted.
```

Comment:
```suggestion
     * @return wrapped_   The amount of wM minted.
```

Reply:
Good catches. Seems like the compiler does not catch this.